SPECS/mariner-rpm-macros/mariner-rpm-macros.spec

```diff
@@ -120,6 +120,9 @@ install -p -m 644 -t %{buildroot}%{rcluadir}/srpm forge.lua
 %{_rpmconfigdir}/macros.d/macros.check
 
 %changelog
+* Fri Oct 15 2021 Ismail Kose <iskose@microsoft.com> - 2.0.7
```

Comment:
the release in this spec file should be updated to 7 as well

Reply:
done